Makefile

```diff
@@ -6,20 +6,26 @@ PI_VERSION=${VERSION}
 
 info:
 	@echo "buildrpm          - build a new RPM from python package index"
+	@echo "signrpm           - sign the RPMs"
 	@echo "fill-devel-repo   - put the newly built packages into the local DEVEL repo"
 	@echo "fill-release-repo - put the newly built packages into the local release repo"
 	@echo "make-repo         - fetch existing repo and build a new local repository with new packages"
 	@echo "push-repo         - push the devel and productive repo to lancelot"
 
-buildrpm:
-	PI_VERSION=${PI_VERSION} rpmbuild --define "_topdir `pwd`" --sign -ba SPECS/privacyidea.spec
-	PI_VERSION=${PI_VERSION} rpmbuild --define "_topdir `pwd`" --sign -ba SPECS/privacyidea-server.spec
+buildrpm: buildpi buildradius buildoracle
```

Comment:
The problem is that currently the documentation tells you to do a ``buildrpm``.
But: The buildoracle probably will never be run again in the next 2 years ;-)

So the build process changes a lot. We would have to change the docs, too.
I am not sure, if changing the buildrpm directive is a good idea.

Reply:
will change that

Reply:
fixed in ffdb7b0